tests/test_fem.py

```diff
@@ -199,23 +199,32 @@ def test_maxvolume(fem_material, show_viewer, box_obj_path):
 @pytest.fixture(scope="session")
 def fem_material_linear():
     """Fixture for common FEM linear material properties"""
-    return gs.materials.FEM.Elastic()
+    return gs.materials.FEM.Elastic(friction_mu=0.5)
 
 
 @pytest.mark.parametrize("backend", [gs.cpu])
-def test_multiple_fem_entities_implicit(fem_material_linear, show_viewer):
+def test_sphere_fall_implicit(fem_material_linear, show_viewer):
     """Test adding multiple FEM entities to the scene"""
     scene = gs.Scene(
         sim_options=gs.options.SimOptions(
-            dt=1e-2,
+            dt=1 / 60,
+            substeps=5,
         ),
         fem_options=gs.options.FEMOptions(
             use_implicit_solver=True,
+            n_newton_iterations=1,
```

Comment:
Do not use custom low-level solver options. Unit tests are meant to validate the default options, because it is what matters for most users. If this test cannot validate the behaviour for the default options, then add another test. (of course not `damping_alpha=0.1`, `damping_beta=0.01`)

Reply:
done